services/sensors/server_test.go

```diff
@@ -33,6 +32,7 @@ func TestServerGetSensors(t *testing.T) {
 		sMap := map[resource.Name]sensors.Service{}
 		server, err := newServer(sMap)
 		test.That(t, err, test.ShouldBeNil)
+		//nolint:staticcheck
```

Comment:
this seems to be necessary as `make build-go lint-go` removes the top-most `//nolint:staticcheck` and then complains.
<img width="1329" alt="Screenshot 2024-05-14 at 1 15 32 PM" src="https://github.com/viamrobotics/rdk/assets/46872531/19f0842b-82d9-466c-8752-7b61fa0cf72f">